content/testing-yaml/testing.md

```diff
@@ -277,4 +277,39 @@ scripts:
 
 {{<notebox>}}
 Flutter for web is available in Flutter version **2.0.0** or higher on the **stable** channel.
-{{</notebox>}}
\ No newline at end of file
+{{</notebox>}}
+
+## Firebase Test Lab
```

Comment:
I think this section should be generic (and not specific to Flutter) for using Firebase Test Lab with any kind of supported tests